odl/discr/partition.py

```diff
@@ -480,19 +492,7 @@ def __getitem__(self, indices):
             min_pt=[-1.0, 1.0, 4.0, 2.0], max_pt=[3.0, 6.0, 5.0, 7.0]
         )
 
-        Indexing picks out sub-intervals (compare with the boundary
-        vectors):
-
-        >>> part[0, 0, 0, 0]
-        nonuniform_partition(
-            [-1.0],
-            [2.0],
-            [5.0],
-            [2.0],
-            min_pt=[-1.0, 1.0, 4.0, 2.0], max_pt=[-0.5, 3.0, 5.0, 3.0]
-        )
```

Comment:
This now became a uniform partition and printed out an enormous line, had to remove it.